pkg/allocator/allocator.go

```diff
@@ -953,11 +953,18 @@ func (a *Allocator) WatchRemoteKVStore(ctx context.Context, remoteName string, r
 	return rc, nil
 }
 
-// RemoveRemoteKVStore removes any reference to a remote allocator / kvstore.
+// RemoveRemoteKVStore removes any reference to a remote allocator / kvstore, emitting
+// a deletion event for all previously known identities.
 func (a *Allocator) RemoveRemoteKVStore(remoteName string) {
```

Comment:
What happens during a temporary controlplane outage? Will this code remove all identities of the remote cluster? I'm thinking in particular about network flapping issues.